docs/reference/setup/install/security-files-reference.asciidoc

```diff
@@ -16,3 +16,20 @@ Keystore that contains the key and certificate for the HTTP layer for this node.
 `transport.p12`::
 Keystore that contains the key and certificate for the transport layer for all
 the nodes in your cluster.
+
+The PKCS#12 keystores `http.p12` and `transport.p12` are password protected and
+the passwords are stored as <<secure-settings,secure settings>>. You can retrieve
+the passwords so that you can inspect or change the keystore contents using
+`bin/elasticsearch-keystore`.
```

Comment:
Some non-blocking wording suggestions to avoid passive voice:
```suggestion
`http.p12` and `transport.p12` are password-protected PKCS#12 keystores. {es}
stores the passwords for these keystores as  <<secure-settings,secure
settings>>. To retrieve the passwords so that you can inspect or change the
keystore contents, use the
<<elasticsearch-keystore,`bin/elasticsearch-keystore`>> tool.
```